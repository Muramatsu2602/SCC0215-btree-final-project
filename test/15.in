<<<<<<< HEAD
8 bin/linha3.bin 4
=======
8 bin/linha.bin 4
>>>>>>> 43edd920be709102cdc4a9bad0450d8d0992194b
691 "S" "CAXIMBA-OLARIA" "LARANJA"
919 "N" "SITIO CERCADO (HORARIO)" "VERDE"
990 NULO "JOSE CULPI-VENEZA" "TURISMO"
950 "F" NULO "VERMELHA"
