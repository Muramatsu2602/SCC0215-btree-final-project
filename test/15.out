5370.170000
